src/strategies/bluesky.js

```diff
@@ -416,17 +446,12 @@ export class BlueskyStrategy {
 
 	/**
 	 * Calculates the length of a message according to Bluesky's algorithm.
-	 * All URLs are counted as 27 characters, all other Unicode characters as is.
+	 * All Unicode characters are counted as is, including URLs at their full length.
 	 * @param {string} message The message to calculate the length of.
 	 * @returns {number} The calculated length of the message.
 	 */
 	calculateMessageLength(message) {
-		// Replace URLs with 27 characters (Bluesky's t.co-like length)
-		const urlAdjusted = message.replace(
-			/https?:\/\/[^\s]+/g,
-			"x".repeat(27),
-		);
-		return [...urlAdjusted].length;
+		return [...message].length;
```

Comment:
We should still count URLs as 27 characters because this represents the length of the text after we do the URL truncation.